### PR TITLE
Linty

### PR DESCRIPTION
**Pull request by bendecoste**

### CODE REVIEW

File: tensorflow_encrypted/protocol/__init__.py [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+                 traceback: Optional[TracebackType]) -> Optional[bool]:
         set_protocol(None)
-
-# from unencrypted_native import UnencryptedNative
-# from unencrypted_fixedpoint import UnencryptedFixedpoint
-from .pond import Pond
-from .securenn import SecureNN
-# from secureml import SecureML
\ No newline at end of file
+        return None
```
[the remark this answers is not in the document]
Comment (jvmancuso) [reply tensorflow_encrypted/protocol/__init__.py]: how about just `return`?

[the remark this answers is not in the document]
Comment (bendecoste) [reply tensorflow_encrypted/protocol/__init__.py]: doesn't like that either :(

**Comment by mortendahl** [comment]
would it be worth having all the tf stubs in a separate module that can be reused elsewhere?

**Comment by bendecoste** [comment]
Do you mean in a different repo? Eventually yes but we need to change them
so often I think it makes sense to put them here for now

**Pull request merged**